Makefile

```diff
@@ -10,4 +10,4 @@ help: ## list makefile targets
 
 .PHONY: build
 build: ## build golang binary
-	go build -o $(PROJECT_NAME) example/config.go example/issue_comment.go example/main.go
\ No newline at end of file
+	go build -o $(PROJECT_NAME)
```

Comment:
why did we not replace it with `... config.go issue_comment.go main.go`?

Reply:
if all required source code files are located in the root of the repo, there's no need to specify them